### DISCUSSION TITLE
提交了一次pr后fork的项目每天都触发action且执行失败

### DISCUSSION THREAD

**Discussion by b1060t** [xmake-io/xmake-repo#921] [Q&A] [answered]
<img width="696" alt="微信截图_20220119093412" src="https://user-images.githubusercontent.com/13270130/150046908-02cc33e0-fe60-4428-b28b-c157bbd34a48.png">

之前提了一次pr之后ci每天都会运行然后失败。不知道具体是什么原因？如何解决？

**Comment by waruqi** [comment, marked as answer]
xmake-repo 仓库 ci 每天会定期执行一次 monkey job 去尝试随机构建几个包，然后遇到失败的，我有时间的时候 偶尔会去看看，然后去修复它们。。

你 fork 了，那整个 ci 也带进来了，这不是 bug ，如果你自己仓库不想也一直跑 可以关了 ci ，或者 master 分支删了 ci 脚本就行了

**Reply by b1060t** [reply]
好的，谢谢！
